packages/perseus/src/components/math-input.tsx

```diff
@@ -66,6 +66,20 @@ type Props = {
      */
     buttonsVisible?: ButtonsVisibleType;
     analytics: PerseusDependenciesV2["analytics"];
+    /**
+     * Optional property to enable the portal functionality for MathInput.
+     * This is very handy in cases where the Popover can't be easily
+     * injected into the DOM structure and requires portaling to
+     * the trigger location.
+     *
+     * Set to "true" by default.
+     *
+     * CAUTION: Turning off portal could cause some clipping issues
+     * especially around legacy code with usage of z-indexing,
+     * Use caution when turning this functionality off and ensure
+     * your content does not get clipped or hidden.
+     */
+    portal?: boolean;
```

Comment:
I could totally be wrong, but I think at this point the only component that uses `perseus/src/components/math-input` is Expression. So a different approach would be to remove this prop and hard-code it to `true` on L348.